Userland/Libraries/LibVideo/VP9/Decoder.cpp

```diff
@@ -12,6 +12,10 @@
 #include "Decoder.h"
 #include "Utilities.h"
 
+#if defined(AK_COMPILER_GCC)
```

Comment:
I _think_ Clang might understand this as well (or at least silently ignores it, so the if around the def is not necessary either way).

Reply:
Interesting, let me try that in a bit. I just copied it from Painter, or some related file at least.

Reply:
Looks like we have error on unknown pragma enabled, and Clang doesn't know what to do with it.

```
/home/zaggy/serenity/Userland/Libraries/LibVideo/VP9/Parser.cpp:16:13: error: unknown pragma ignored [-Werror,-Wunknown-pragmas]
#pragma GCC optimize("O3")
```